### ISSUE TITLE
standalone tests failing locally due to 32bit libs being present on a 64 bit machine

### ISSUE THREAD

**Issue by jayvdb** [Nuitka/Nuitka#2011]
Tests failing locally due to 32bit libs being present on a 64 bit machine.

```
Run '/usr/bin/python3 ./tests/standalone/run_all.py search' in '/home/jayvdb/python/Nuitka'.
Using concrete python 3.10.9 on x86_64
Consider output of standalone mode compiled program: BrotliUsing.py
Comparing output of 'BrotliUsing.py' using '/usr/bin/python3' with flags silent, expect_success, --standalone, remove_output, cpython_cache, timing, --nowarn-mnemonic=debian-dist-packages ...
--- /usr/bin/python3 (stderr)
+++ nuitka (stderr)

@@ -0,0 +1,2 @@

+/usr/lib64/gcc/x86_64-suse-linux/12/../../../../x86_64-suse-linux/bin/ld: skipping incompatible /usr/lib/libgcc_s.so.1 when searching for libgcc_s.so.1
+/usr/lib64/gcc/x86_64-suse-linux/12/../../../../x86_64-suse-linux/bin/ld: skipping incompatible /usr/lib/libgcc_s.so.1 when searching for libgcc_s.so.1
```

I have a patch coming for this.

**Comment by kayhayen** [comment]
This should be done in output comparison I think, matching a regexp of this kind of line.

**Comment by jayvdb** [comment]
I think that is what I did in the PR ?

**Comment by kayhayen** [comment]
I will make this part of 1.4.1 when it comes

**Comment by kayhayen** [comment]
This was released, please use 1.4.3 though, there was an important incorrect code execution bug fixed in there.
